svr-main.c

```diff
@@ -315,10 +315,8 @@ static void main_noinetd() {
 				getaddrstring(&remoteaddr, NULL, &remote_port, 0);
 				dropbear_log(LOG_INFO, "Child connection from %s:%s", remote_host, remote_port);
 
-				#if DROPBEAR_RESTRICT_FIXED_HOST_IP
-				#define MAX_IPV4_LEN 9
-				printf("%s\n%s\n", remote_host, svr_opts.allowed_host_ip_addr);
-				if (strcmp(remote_host, svr_opts.allowed_host_ip_addr))
+				#if DROPBEAR_RESTRICT_FIXED_CLIENT_IP
```

Comment:
Move this to a validate_client_ip(remote_host) function,
provide two versions depending on the macro.

Reply:
Done